src/main/resources/templates/index.html

```diff
@@ -12,21 +12,9 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
           integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 
-    <style>
-        body {
-            min-height: 75rem;
-            padding-top: 4.5rem;
-        }
-    </style>
+    <link rel="stylesheet" type="text/css" href="css/cite.css">
+    <script src="js/itemScript.js" type="text/javascript"></script>
```

Comment:
removed the JavaScript and the in-line styles and referred to them here.

Reply:
Good.